### PR TITLE
CORTX-32899: New jenkins job to install and run predefined workloads

### PR DESCRIPTION
**Pull request by rahul27kumar**
Signed-off-by: Rahul Kumar <rahul.kumar@seagate.com>

# Problem Statement
- Added new jenkins jon to install perfline and trigger predefined workloads

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by shailesh-vaidya — COMMENTED** [review]
Please add code to https://github.com/Seagate/cortx-re/tree/main/jenkins/performance-ci 
Please address Codacy issues - https://github.com/Seagate/cortx-re/pull/1174/checks?check_run_id=7851581701 
Please add Jenkins job used for testing in PR template

File: solutions/kubernetes/cortx-perfline.sh [hunk 1]
```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```
Comment (shailesh-vaidya) [inline solutions/kubernetes/cortx-perfline.sh:1]: Please move scripts to https://github.com/Seagate/cortx-re/tree/main/scripts/performance

File: solutions/kubernetes/install-perfline.sh [hunk 2]
```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -x
```
Comment (shailesh-vaidya) [inline solutions/kubernetes/install-perfline.sh:1]: Please move scripts to https://github.com/Seagate/cortx-re/tree/main/scripts/performance

File: solutions/kubernetes/install-perfline.sh [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```
Comment (shailesh-vaidya) [inline solutions/kubernetes/install-perfline.sh:3]: Please replace 2021 to 2022

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

**Comment by rahul27kumar** [comment]
jenkins job: https://eos-jenkins.colo.seagate.com/job/Motr/job/cortx-perfline/19/console

**Review by gauravchaudhari02 — APPROVED** [review]

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**